code/__DEFINES/~skyrat_defines/traits.dm

```diff
@@ -60,3 +60,6 @@
 #define TRAIT_XENO_ABILITY_GIVEN "xeno_ability_given"
 /// Determines if something can receive healing from a xeno
 #define TRAIT_XENO_HEAL_AURA "trait_xeno_heal_aura"
+
+/// Whether something is repairable by the anvil
+#define TRAIT_ANVIL_REPAIR (1<<0)
```

Comment:
Traits are a specific concept using `ADD_TRAIT`, `REMOVE_TRAIT`, etc. This should probably be in `code\__DEFINES\~skyrat_defines\obj_flags.dm` and not have a `TRAIT_` prefix as that is misleading.

Preferably IHMO there should be a unique new prefix for flags going into `skyrat_obj_flags`. I'll let you decide regarding that.

Reply:
I think I'm going to keep the normal naming sense-- I did remove the TRAIT portion though as you are right.